config/routes.rb

```diff
@@ -0,0 +1,19 @@
+Scribble::Application.routes.draw do
+  root "posts#index"
+  resources :posts, only: [:index, :show, :new]
+  resources :comments, only: [:index, :show, :new]
```

Comment:
So how do I go about accessing a comment(s) that belongs to a particular post? Can we accomplish that with the way your routes and controllers are currently set up?